common-app-lib/src/cmr/common_app/services/search/parameter_validation.clj

```diff
@@ -229,7 +260,7 @@
 
 (def standard-valid-params
   "The set of standard valid query level parameters."
-  #{:page-size :page-num :offset :sort-key :result-format :options})
+  #{:offset :options :page-num :page-size :result-format :scroll :scroll-id :sort-key})
```

Comment:
Wait, forget that. I am reading it from the header and adding it to the params in `process-params` in the search app routes.clj.